theme_paptic/static/src/js/tour.js

```diff
@@ -1,11 +1,5 @@
-odoo.define("theme_paptic.tour.paptic", function (require) {
-"use strict";
-
-const core = require("web.core");
-const _t = core._t;
-
-const wTourUtils = require("website.tour_utils");
-var tour = require("web_tour.tour");
+/** @odoo-module */
+import wTourUtils from 'website.tour_utils';
```

Comment:
I know I am annoying but if you can add a blank line after `/** @odoo-module */`... thanks :+1: